_news/2018-10-08-forseti-cai.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Asset Inventory
+author: Andrew Hoying
+---
+With the release of [asset inventory](https://www.google.com/url?q=https://cloud.google.com/blog/products/gcp/gain-insights-about-your-gcp-resources-with-asset-inventory&sa=D&source=hangouts&ust=1539109101901000&usg=AFQjCNGWNRSXziSyWifOvmUnu44CyDuI6Q), Google Cloud Platform now provides a
```

Comment:
Let's give a shorter direct url, rather than one that is from a google.com query.

Reply:
Updated to direct URL.